src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NSDbSinkWriter.scala

```diff
@@ -42,7 +45,10 @@ class NSDbSinkWriter(connection: NSDB,
                      defaultValue: Option[java.math.BigDecimal],
                      retentionPolicy: Option[Duration],
                      shardInterval: Option[Duration],
-                     semanticDelivery: Option[SemanticDelivery])
+                     semanticDelivery: Option[SemanticDelivery],
+                     retries: Int,
```

Comment:
I prefer to remote Option because in this case what I did is to getOrElse with the default value and so I avoid to import the NSDBConfig file again here

Reply:
It's fine